### DISCUSSION TITLE
Is there a font requirement for neorg?

### DISCUSSION THREAD

**Discussion by jrmallorca** [nvim-neorg/neorg#59] [Q&A] [answered]
Hello, I was wondering if there was a font requirement for neorg as there are a few wrongly displayed graphics in my .norg files compared to the pictures/gifs in the main page. The following picture should demonstrate it.

Heading works fine but list items aren't recognised and modifiers don't work.

![image](https://user-images.githubusercontent.com/48072014/128624624-57c7706a-556f-4de0-9363-73f7eeb83885.png)

**Comment by vhyrro** [comment, marked as answer]
We don't have any specific font requirements other than having some nerd fonts installed in your system - that should suffice from what i know. As for that bold and underline issue are you running the unstable branch?

**Reply by jrmallorca** [reply]
Installing nerd fonts fixed the list item issue, thanks.

I'm most likely not running the unstable branch as I have no idea how to know or specify how to do so. Would that be the branch where those modifiers are working?

**Reply by vhyrro** [reply]
Yeah, all the modifiers are highlighted in the unstable branch via regex highlighting. I'm almost done with the new treesitter parser so I'll use that for highlighting when we get to it. Your plugin manager most definitely has an option for specifying the branch to use so go check that out :)

**Comment by kanodiaayush** [comment]
I have the same issue but I don't get it resolved after adding nerd fonts.

**Reply by vhyrro** [reply]
Hmm, you gotta make sure that your terminal is also using those nerd fonts in order for them to properly work most of the time. Have you tried that?

**Reply by kanodiaayush** [reply]
Thank you; that worked. I was ssh ing from my mac into a linux server and I need to change terminal font on my mac to a nerd font.
